packages/framework-types/src/envelope.ts

```diff
@@ -68,14 +68,31 @@ export interface ReadModelEnvelope {
   value: ReadModelInterface
 }
 
+export interface ReadModelResult {
```

Comment:
I think the name `ReadModelListResult` would be more accurate as if not, it seems this is a result of only one read model. 
Not a hard opinion though ;-)